src/Misc/Hooks.MapTint.cpp

```diff
@@ -0,0 +1,82 @@
+#include <Utilities/Macro.h>
+#include <Helpers/Macro.h>
+#include <ScenarioClass.h>
+#include <Utilities/TemplateDef.h>
+
+namespace MapTintFix
+{
+	std::unique_ptr<LightConvertClass> TiberiumLightDrawer;
```

Comment:
`LightConvertClass` has a virtual destructor, you might end up allocating with dll new but deallocating with exe delete. Besides unique_ptr does not seem to increase safety in this case.
```suggestion
	LightConvertClass* TiberiumLightDrawer;
```